pontoon/checks/libraries/pontoon_db.py

```diff
@@ -65,6 +66,17 @@ def run_checks(entity, string):
                     'Translation too long'
                 )
 
+    # Bug 1599056: Original and translation must either both end in a newline,
+    # or none of them should.
+    if resource_ext == 'po':
+        if (
+            original.endswith('\n') and not string.endswith('\n') or
+            not original.endswith('\n') and string.endswith('\n')
```

Comment:
I think this is easier written as `original.endswith('\n') == string.endswith('\n')`?

Reply:
Ohhhhhhhh. :-) `!=` instead of `==`, but yeah. Much easier.